File: 1060.cpp
#include <iostream>
using namespace std;

int main() {
    int c;
    double a[6];
    
    for(int i = 0; i < 6; i++) {
        cin >> a[i];
        if(a[i] > 0)
            c++;
    }
    cout << c << " valores positivos" << endl;
    return 0;
}

File: Iniciante/C#/1060.cs
using System;
class URI {
    static void Main(string[] args) { 
        int c = 0;
        for(int i = 0; i < 6; i++) {
            double a = double.Parse(Console.ReadLine());
            if(a > 0) c++;
        }
        Console.WriteLine(c + " valores positivos");
    }   
}